Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -836,7 +836,7 @@ static void WriteStage(T& out, pixel_shader_uid_data* uid_data, int n, API_TYPE
 		uid_data->SetTevindrefTexmap(i, texmap);
 
 		out.Write("\ttextemp = ");
-		SampleTexture<T>(out, "float2(tevcoord.xy)", texswap, texmap, ApiType);
+		SampleTexture<T>(out, "(float2(tevcoord.xy) + 0.05)", texswap, texmap, ApiType);
```

Comment:
It was the lowest value I could find that fixed the issue. Anything lower and it only partially fixed it.